pkg/archive/untar_nested.go

```diff
@@ -0,0 +1,129 @@
+package archive
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+// Untar reads the gzip-compressed tar file from r and writes it into dir.
+func UntarNested(r io.Reader, dir string) error {
+	return untarNested(r, dir)
+}
+
+func untarNested(r io.Reader, dir string) (err error) {
```

Comment:
Should we move this functions under `pkg/archive/untar.go`?
I think it makes much more sense.

Reply:
No I don't see any reason to have one file for this.